src/Sulu/Bundle/MediaBundle/Resources/public/js/services/media-router.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(['services/husky/mediator'], function(mediator) {
+
+    'use strict';
+
+    var instance = null;
+
+    /** @constructor **/
+    function AccountRouter() {
+    }
+
+    AccountRouter.prototype = {
+
+        /**
+         * Navigates to collection view of given collectionId
+         * @param collectionId
+         */
+        toCollection: function(collectionId) {
+            if (!!collectionId) {
+                mediator.emit('sulu.router.navigate', 'media/collections/edit:' + collectionId + '/files', true, true);
```

Comment:
@marcelmoosbrugger a reload of the current page is required on collection-move, collection-change or language change, the parameters are correct